packages/web/src/components/add-cash/CashBalanceSection.tsx

```diff
@@ -20,8 +25,25 @@ export const CashBalanceSection = ({ balance }: CashBalanceSectionProps) => {
         <Flex alignItems='center' gap='s'>
           <IconLogoCircleUSDC />
           <Text variant='heading' size='s' color='subdued'>
-            {messages.cashBalance}
+            {walletMessages.cashBalance}
           </Text>
+          <Tooltip
```

Comment:
you can just use `TooltipInfoIcon.tsx` here

Reply:
tooltip was getting clipped by modal boundaries when i tried this, so just gonna leave as is.